modules/preloader/browser.js

```diff
@@ -28,6 +28,7 @@ process.on('loaded',function () {
 
 
 window.mist = mist();
+window.mistMode = remote.getGlobal('mode');
```

Comment:
the mistMode shouldn't be exposed to dapps in browser.js.
We also don't want to require remote her, for security reasons.

The issue should be fixed in the wallet.js